src/index.php

```diff
@@ -38,4 +38,5 @@
     <body>
         <?= $content ?>
     </body>
+    <script src="./main.js"></script>
```

Comment:
Wrong path, you need to move on `./js/main.js`
```suggestion
    <script src="./js/main.js"></script>
```

Reply:
fixed